code/game/mecha/combat/fighter.dm

```diff
@@ -171,7 +171,10 @@
 
 /obj/mecha/combat/fighter/proc/consider_gravity(var/moved = FALSE)
 	var/gravity = has_gravity()
-	if(gravity && ground_capable && occupant)
+	if(gravity && !stabilization_enabled)
+		step_in = 10 //Slow now that they're on landing gear
```

Comment:
as far as I can tell you are not restoring the previous step_in, so once they were on landing gear they can never regain their original speed